example/anisotropic_diffusion.cpp

```diff
@@ -0,0 +1,99 @@
+#include <boost/gil/algorithm.hpp>
+#include <boost/gil/typedefs.hpp>
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/io/typedefs.hpp>
+#include <boost/gil/image_processing/numeric.hpp>
+
+#include <cmath>
+#include <string>
+#include <vector>
+#include <type_traits>
```

Comment:
Nitpick: [includes need to be ordered](https://github.com/boostorg/gil/wiki/Include-Directives-Order)